vignettes/implementing-cookies.Rmd

```diff
@@ -0,0 +1,110 @@
+---
+title: "Implementing cookie consent"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Implementing cookie consent}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+Cookie consent is a mandatory requirement for any dashboard when using cookies 
```

Comment:
This sentence is a bit clunky I think? Also I know most of us know what cookies are but is it worth a one sentence explanation of what they are/do?